pkg/api/azenvtypes.go

```diff
@@ -80,6 +81,22 @@ var (
 		VnetCNILinuxPluginsDownloadURL:   "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-linux-amd64-" + AzureCniPluginVerLinux + ".tgz",
 		VnetCNIWindowsPluginsDownloadURL: "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-windows-amd64-" + AzureCniPluginVerWindows + ".zip",
 		ContainerdDownloadURLBase:        "https://storage.googleapis.com/cri-containerd-release/",
+		ImageRepoOverrides: map[string]ImageRepoOverride{
+			"k8s.gcr.io/hyperkube-amd64": {
+				Registry: "mcr.microsoft.com",
+				Repo:     "k8s/core/hyperkube-amd64",
+				// TODO: Setup tag overrides so we can map upstream versions to Azure k8s versions
+				// For now just using the upstream version works fine.
+			},
+			"k8s.gcr.io/pause-amd64": {
+				Registry: "mcr.microsoft.com",
```

Comment:
So Im leaning towards keeping this in mcr unless someone has a strong objection to that?